source/docs/guides/apple_pay/index.md

```diff
@@ -161,6 +176,25 @@ func paymentAuthorizationViewController(_ controller: PKPaymentAuthorizationView
 }
 ```
 
+>It's very important to notice that the ***payment_token is Base64 encoded*** including the paymentData.
```

Comment:
Change to:
"It's important that the ***payment_token is Base64 encoded*** and included in the paymentData object."

Reply:
Done!